app/inputs/search_select_input.rb

```diff
@@ -10,5 +10,4 @@ def input_html_options
     opts["data-selected"] = relation.try(opts["data-display_name"].to_sym)
     super.merge opts
   end
-end
-
+end
```

Comment:
@victor95pc you can use [git amend](https://es.atlassian.com/git/tutorials/rewriting-history/git-commit--amend/) to add the new line and please the monkey :smile: